testsuite/integration-arquillian/tests/base/pom.xml

```diff
@@ -852,6 +852,7 @@
                                 <keycloak.authorizationCache.enabled>false</keycloak.authorizationCache.enabled>
                                 <keycloak.realmCache.enabled>false</keycloak.realmCache.enabled>
                                 <keycloak.userCache.enabled>false</keycloak.userCache.enabled>
+                                <keycloak.userSessionPersister.provider></keycloak.userSessionPersister.provider>
```

Comment:
Why is this needed?

Reply:
I would just expect this is not necessary as `userSessionPersister` is disabled by the `isSupported` method. Should this be done also for?:
https://github.com/keycloak/keycloak/blob/7d96f3ad5ad32c4a78a9d571ad891ae953258332/testsuite/integration-arquillian/tests/base/src/test/resources/META-INF/keycloak-server.json#L180-L182

Reply:
We decided to investigate `userFederatedStorage` in a separate issue https://github.com/keycloak/keycloak/issues/12895